Cython/Debugger/Cygdb.py

```diff
@@ -104,42 +104,36 @@ def make_command_file(path_to_debug_info, prefix_code='',
 
     return tempfilename
 
-usage = "Usage: cygdb [options] [PATH [-- GDB_ARGUMENTS]]"
 
-def main(path_to_debug_info=None, gdb_argv=None, no_import=False):
+def main():
```

Comment:
These were removed because they are superfluous. `path_to_debug_info` is replaced by `--build_dir`, `gdb_argv` is replaced by ``gdb_argv` and `no_import` is inferred from the other options

Reply:
I also don't think these were used at all? If you look at `bin/cygdb.py` is simply invokes this module without arguments:

```python
import sys

from Cython.Debugger import Cygdb as cygdb

if __name__ == '__main__':
    cygdb.main()
```